01_fib/NicholeL/fib.go

```diff
@@ -0,0 +1,49 @@
+package main
+
+import (
+	"fmt"
+	"io"
+	"os"
+)
+
+var out io.Writer = os.Stdout
+
+func abs(args int) (res int) {
+	if args == 0 {
+		return 0
+	}
+	if args > 0 {
+		return args
+	}
+	return args * (-1)
+
+}
+
+func fib(args int) {
+	a, b := 0, 1
+	if abs(args) > 92 {
+		fmt.Fprint(out, "Fibonacci numbers greater than 92 not supported")
+		return
+	}
+	if abs(args) <= 1 {
+		fmt.Fprintln(out, args)
```

Comment:
return here rather than `if`.
Deep nesting levels are harder to comprehend.

Reply:
yep, thanks for your advice, will update to return